readme.md

```diff
@@ -181,6 +181,29 @@ test(function (t) {
 });
 ```
 
+### Generators support
+
+AVA supports [generator functions](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/function*) and  out-of-the-box. 
+
+```js
+test(function * (t) {
+	const value = yield generatorFn();
+	
+	t.end();
+});
+```
+
+### async/await support
+
+AVA also supports [async/await](https://github.com/yortus/asyncawait) with no configuration required.
+
+```js
+test(async function (t) {
```

Comment:
I was wrong, you can use arrow function with async. Can you add an additional example using arrow functions? https://tc39.github.io/ecmascript-asyncawait/#async-arrows